ush/machine/odin.yaml

```diff
@@ -9,8 +9,11 @@ platform:
   QUEUE_FCST: workq
   PARTITION_HPSS: workq
   QUEUE_HPSS: workq
+  PARTITION_PRDGEN: workq
+  QUEUE_PRDGEN: workq
```

Comment:
Would it work for the prdgen task to use `QUEUE_` and `PARTITION_DEFAULT` instead of introducing a new queue? Are there platforms that may need a totally separate queue to do the thing for this job that's different than others?

I think we've accumulated 3 so far for a catchall (DEFAULT), the forecast (because we may want beefier nodes than a default on the cloud, or on Jet), and then HPSS for access nodes that have connectivity to the internet.

Reply:
Yes I think it should.  I will use QUEUE_DEFAULT and PARTITION_DEFAULT for this task.